OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -164,10 +164,7 @@ void SyncMultiplayerCount()
 				var index = int.Parse(name.Substring(5));
 
 				if (index >= newCount)
-				{
 					Players.Players.Remove(name);
-					worldRenderer.World.Players.RemoveAll(pp => pp.InternalName == name);
```

Comment:
@Mailaender: When opening the editor, `worldRenderer.World.Players` is always an empty collection and thus this line was a no-op. So I removed this line as part of my other changes. Just wanted to bring it to your attention in case this indicates a possible bug or anything.